### PR TITLE
Adding new Bynder logo

### PR DESCRIPTION
**Pull request by webknit**
- Adding new SVG
- Restricting max width specifically for bynder logo
- Applying border radius

### CODE REVIEW

**Comment by AmeeMorris** [comment]
Whats your plan to featureflag this? might be worth having both in GUI then we can pass in a prop (controlled by a flag) from webapp that will switch between the two

File: lib/Logo/index.tsx [hunk 1]
```diff
@@ -9,11 +9,12 @@ import logoSVG from "../../assets/logo.svg";
 
 export function Logo(props: any) {
   const LogoPath = props.url || logoSVG;
+  console.log(LogoPath);
```
Comment (AmeeMorris) [inline lib/Logo/index.tsx:12]: log

File: lib/LogoGC/index.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ */
+
+export function LogoGC(props: any) {
+  const LogoPath = props.url || logoSVG;
+  const image =
+    typeof LogoPath === "string" ? (
+      <img src={props.url} alt={props.alt} className="logo__image" />
+    ) : (
+      <span className="logo__image">
+        <LogoPath />
```
Comment (AmeeMorris) [inline lib/LogoGC/index.tsx:17]: This points to the same svg as the new bynder one, we need the old svg in place :)

Comment (webknit) [reply lib/LogoGC/index.tsx:17]: Added old SVG back and also added a new story so it's clear what's what.

**Review by timbryandev — APPROVED** [review]

**Pull request merged**